scalableminds/webknossos
# Deactivate all dataSources, that are no longer available on a datasto…

### Mailable description of changes
- Datasets removed from a dataStore will automatically be deactivated and marked as no longer available.

### Steps to test:
- remove a previously existing and reported (shows up on dashboard) dataset
- trigger an inbox check `/data/triggers/checkInbox`
- the removed dataset should no be labeled as unavailable in the dashboard

### Issues:
- fixes #1898

------
